redfish/provider/resource_redfish_user_account.go

```diff
@@ -135,6 +139,70 @@ func (*UserAccountResource) Schema(_ context.Context, _ resource.SchemaRequest,
 				Optional:            true,
 				Computed:            true,
 			},
+			"users": schema.ListNestedAttribute{
+				Description:         "Options to enable or disable the boot device.",
+				MarkdownDescription: "Options to enable or disable the boot device.",
+				Optional:            true,
+				Validators: []validator.List{
+					listvalidator.ConflictsWith(tfpath.Expressions{
+						tfpath.MatchRoot("user_id"),
+						tfpath.MatchRoot("username"),
+						tfpath.MatchRoot("password"),
+						tfpath.MatchRoot("role_id"),
+						tfpath.MatchRoot("enabled"),
+					}...),
+				},
+				NestedObject: schema.NestedAttributeObject{
+					Attributes: map[string]schema.Attribute{
+						"user_id": schema.StringAttribute{
+							MarkdownDescription: "The ID of the user. Cannot be updated.",
+							Description:         "The ID of the user. Cannot be updated.",
+							Optional:            true,
+							Computed:            true,
+						},
+						"username": schema.StringAttribute{
+							MarkdownDescription: "The name of the user",
+							Description:         "The name of the user",
+							Required:            true,
+							Validators: []validator.String{
+								stringvalidator.LengthBetween(minUserNameLength, maxUserNameLength),
+							},
+						},
+						"password": schema.StringAttribute{
+							MarkdownDescription: "Password of the user",
+							Description:         "Password of the user",
+							Required:            true,
+							Sensitive:           true,
+							Validators: []validator.String{
+								stringvalidator.LengthBetween(minPasswordLength, maxPasswordLength),
+							},
+						},
+						"role_id": schema.StringAttribute{
```

Comment:
Please verify if we want to use API nomenclature here.
Because UI shows `User Role` column and the words `Operator`, `Administrator`, etc. are commonly used user roles.